src/app/cube/usage-stats/top-downloads/top-downloads.component.html

```diff
@@ -2,6 +2,9 @@
     <h2 class="title">Most Downloaded Learning Objects</h2>
     <div class="downloads-container" *ngIf="!loading; else loadingTemplate">
       <div *ngFor="let lo of learningObjects">
+        <div [ngClass]="'length ' + lo.length">
+          {{ lo.length | titlecase }}
+        </div>
```

Comment:
Could we create a wrapper around this so that the titles of the learning objects always line up?

Reply:
I'm probably misunderstanding, is the below commit what you meant?

Reply:
@pmanal1 Can I see the updated screenshot?

Reply:
![Screen Shot 2023-01-19 at 12 29 32 PM](https://user-images.githubusercontent.com/93054689/213517471-7af84122-b8b0-4448-a363-155b0736e1bb.png)

Here's the updated screenshot

Reply:
How does this look on mobile?